yapapi/golem.py

```diff
@@ -172,6 +172,26 @@ def operative(self) -> bool:
         return self._engine.started
 
     async def start(self) -> None:
+        """Start the Golem engine in non-contextmanager mode.
+
+        The default way of using Golem::
```

Comment:
```suggestion
        The default way of using Golem:
```

Reply:
@zakaprov `::` is a special-magic-sphinx-thing https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks that in this scenario renders as a single `:` and also handles the next paragraph (code) in a correct way. Yep, I also love this! :P